AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -4,7 +4,8 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 
-Holds constants used across both hydra and non-hydra scripts.
+Holds script canvas related constants used across both hydra and non-hydra scripts.
+We are deprecating VARIABLE_TYPES in favor of the dictionary since we can access the dictionary nicely with dot notation instead of indices
```

Comment:
Add a deprecation notice for the file in general in favor of the `EditorPythonTestTools/consts/` files

Reply:
Added a notice to the scripting constants file.